src/iOS/Xamarin.iOS/Samples/Security/TokenSecuredKnownUser/TokenSecuredKnownUser.cs

```diff
@@ -191,10 +192,10 @@ private void LayerLoadStatusChanged(object sender, Esri.ArcGISRuntime.LoadStatus
 
                     // Create a credential for this resource.
                     knownCredential = await AuthenticationManager.Current.GenerateCredentialAsync
-                                            (info.ServiceUri,
-                                             username,
-                                             password,
-                                             info.GenerateTokenOptions);
+                    (info.ServiceUri,
```

Comment:
Bizarre code indentation/formatting here. Why?

Reply:
I think I ran a code formatter by accident, although I'm surprised it would have done that. I'll revert before merging.